src/include/stir/ProjDataInfoCylindricalNoArcCorr.inl

```diff
@@ -175,12 +194,13 @@ get_det_pos_pair_for_bin(
   dp.pos1().axial_coord()=a1;
   dp.pos2().tangential_coord()=t2;
   dp.pos2().axial_coord()=a2;
+  dp.timing_pos() = std::abs(bin.timing_pos_num())*this->get_tof_mash_factor();
```

Comment:
https://github.com/ashgillman/STIR/commit/dff2c9fc1923f1839ad879478eb7fcca5283af07#r59620526

> if `timing_pos_num` was negative, shouldn't it have change the rest of the bin then?

I'm not sure why its necessary, but I think by definition this should already be positive in this convention?

Reply:
confusingly, at present `bin.timing_pos_num` can be negative but `detection_pos_posi.timing_pos` not.

Reply:
See https://github.com/NikEfth/STIR/issues/26